lib/email/utils/helpers.js

```diff
@@ -126,6 +126,8 @@ function logAmplitudeEvent (log, message, eventInfo) {
   }, {
     device_id: message.deviceId || getHeaderValue('X-Device-Id', message),
     email_domain: eventInfo.domain,
+    email_sender: message.fxaEmailSender || getHeaderValue('X-Fxa-Email-Sender', message),
+    email_service: message.fxaEmailService || getHeaderValue('X-Fxa-Email-Service', message),
```

Comment:
In isolation, it seems odd that these have an "fxa" prefix while our various other headers do not, but *shrug*...

Reply:
Fwiw, I was just trying to distinguish more clearly from the existing `X-Service-Id` header by calling out that service here is an internal FxA thing. And because they're a pair of values I opted to name the sender one similarily. But I'll take it out if it seems odd.

Reply:
I don't feel strongly either way